yb-voyager/src/srcdb/utils.go

```diff
@@ -26,3 +30,19 @@ func nameContainsCapitalLetter(name string) bool {
 	}
 	return false
 }
+
+func findAllExecutablesInPath(executableName string) ([]string, error) {
+	pathString := os.Getenv("PATH")
+	if pathString == "" {
+		return nil, fmt.Errorf("PATH environment variable is not set")
+	}
+	paths := strings.Split(pathString, ":")
```

Comment:
May be better to use the `os.PathListSeparator` here

Reply:
Done